implementations/rust/ockam/ockam_identity/src/identities/identities.rs

```diff
@@ -157,14 +157,23 @@ impl Identities {
     /// Return a builder for identities with a specific database
     #[cfg(feature = "storage")]
     pub fn create(database: SqlxDatabase) -> IdentitiesBuilder {
+        Self::create_with_node(database, "default")
```

Comment:
I would say this function can be error-prone, it looks like node name value is not required, while it can be very important. Can we delete the function?

Reply:
Unfortunately, this can not be removed without the larger refactoring that I mentioned it the "This is still not great" section.